modules/browser/BrowserCapabilities.js

```diff
@@ -112,6 +113,26 @@ export default class BrowserCapabilities extends BrowserDetection {
             || this.isWebKitBased();
     }
 
+    /**
+     * Returns whether the browser is supported for Android
+     * @returns {boolean} true if the browser is supported for Android devices
+     */
+    isSupportedAndroidBrowser() {
+        return this.isChromiumBased() || this.isFirefox();
+    }
+
+    /**
+     * Returns whether the browser is supported for iOS
+     * @returns {boolean} true if the browser is supported for iOS devices
+     */
+    isSupportedIOSBrowser() {
```

Comment:
What does this return for say Brave on iOS 12?

Reply:
webkit on ios, so it will return true. this is just moved logic from jitsi-meet, and adding the extra safari version check